lib/config/virtual_container_host.go

```diff
@@ -248,6 +253,14 @@ func (t *VirtualContainerHostConfigSpec) IsCreating() bool {
 	return t.ExecutorConfig.ID == CreatingVCH
 }
 
+func (t *VirtualContainerHostConfigSpec) SetGrantPerms() {
+	t.GrantPermsLevel = AddPerms
+}
+
+func (t *VirtualContainerHostConfigSpec) ShouldGrantPerms() bool {
+	return t.GrantPermsLevel != ""
```

Comment:
Based on my understanding, this will always return true with this change, which would mean that `GrantOpsUserPerms` runs for every `configure` operation. Is that valid? I'd also like to see the `vic-machine configure` integration test passing with this change - please run specific ci.

Reply:
In the latest change I removed the flag from being available in vic-machine (both create and configure). I have done that for two reasons: need a better way to handle Boolean flags, lack of Integration tests. 

Eventually it will work this way:

Requesting the grant option in either create or configure, will do the granting and save the value of the option in the config for the VCH. This will be automatically be applied on each of the following configure request unless you explicitly specify --ops-grant-perms=false. Not specifying the flag will leave the situation as it was in the last create/configure operation. This is done to allow the user to omit the option in the following configure operations. Making it somewhat persistent. I have discussed with George at length about the approach and that is what we think is the best choice for now.

Reply:
> Not specifying the flag will leave the situation as it was in the last create/configure operation.

Could you point me to the code where this is handled? Thanks.

Also, if we're not enabling this feature in `vic-machine` yet, would it make sense to set `AddPerms` to `""` so `GrantOpsUserPerms` isn't invoked at all?

Reply:
The code that handles the different flavor of the Bool Flag is not in this commit. That is why I removed its external visibility. To support the three variants:
- the option is not set,
- the option is set,
- the option is set to false,
We need to implement a new version of the Bool flag, that's why I have removed it from now.
Regarding AddPerms I am not sure what you mean. GrantOpsUserPerms() is called only when GrantPermLevel is set to a non empty string. In the current version GrantPermLevel is always the empty string.

Reply:
> In the current version GrantPermLevel is always the empty string.

We're saying `AddPerms = "ADD"` and `t.GrantPermsLevel = AddPerms`, so `ShouldGrantPerms()` will return true. Are we setting `AddPerms = ""` somewhere else?

Reply:
I'm not sure if this is retreading the same ground as above. Was expecting this line to be:
```go
   return t.GrantPermsLevel == AddPerms
```